themes/openSUSE/src/common.inc

```diff
@@ -207,7 +207,7 @@
       pop
 
       main.recalc
-      main.redraw
+      true main.redraw
```

Comment:
I would find it more readable when using a symbol instead of a boolean parameter:
```
/with_boot_options main.redraw
/without_boot_options main.redraw
```
If I understand the language right, that would be translated with this argument parsing code prepended to main.redraw:
```
/with_boot_options eq true false ifelse
```